src/Description.js

```diff
@@ -1,15 +1,45 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './Description.css';
+import Poster from './Poster';
+import Button from './Button';
 
-function Description({title, rating, summary, releaseDate}) {
-  return (
-    <div>
-    <h2>{title}</h2>
-    <p>{releaseDate}</p>
-    <p>{summary}</p>
-    <h3>{rating.toFixed(1)}  ⭐️</h3>
-    </div>
-  )
+class Description extends Component {
+  constructor(){
+    super()
+    this.state = {
+      loading: true,
+      singleMovie: null,
+    }
+  }
+
+  componentDidMount = () => {
+      fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.props.currentFilmId}`)
+      .then(response => response.json())
+      .then(data => this.setState({singleMovie: data.movie, loading: false}))
+      .then(this.render())
```

Comment:
I think we can maybe delete this? Since it didn't fix our timing problem. I think render is called automatically by react.

Reply:
Yes, I will delete it. Good catch!